applications/mne_dipole_fit/main.cpp

```diff
@@ -97,6 +94,11 @@ int main(int argc, char *argv[])
     DipoleFit dipFit(&settings);
     ECDSet set = dipFit.calculateFit();
 
+    ECDView ecdViewer(settings, set);
```

Comment:
how about making here a shared pointer ECDView::SPtr and initializing it whtin the if settings.gui - so the gui gets not initialized when no gui is wanted